platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -11,113 +11,169 @@ For details, visit [Azure Free Account][az-create-account].
 
-Azure uses 'accounts' for each service. For example, an [Azure Storage account][az-learn-storage] will house a collection of blob containers, file shares, queues, and tables. While you can have multiple Azure Storage and Azure Batch accounts in an Azure subscription, each compute environment on the platform can only use one of each (one storage and one Batch account). You can set up multiple compute environments on the platform with different credentials, storage accounts, and Batch accounts.
+#### Resource groups
 
-### Resource group
+An Azure resource group is a logical container that holds related Azure resources such as virtual machines, storage accounts, databases, and more. A resource group serves as a management boundary to organize, deploy, monitor, and manage the resources within it as a single entity. Resources in a resource group share the same lifecycle, meaning they can be deployed, updated, and deleted together. This also enables easier access control, monitoring, and cost management, making resource groups a foundational element in organizing and managing cloud infrastructure in Azure.
 
-To create Azure Batch and Azure Storage accounts, first create a [resource group][az-learn-rg] in your preferred region.
+#### Accounts
 
-:::note
-A resource group can be created while creating an Azure Storage Account or Azure Batch account.
-:::
+Azure uses accounts for each service. For example, an [Azure Storage account][az-learn-storage] will house a collection of blob containers, file shares, queues, and tables. An Azure subscription can have multiple Azure Storage and Azure Batch accounts. However, a Platform compute environment can only use one of each (one Storage and one Batch account). Multiple Platform compute environments can be set to use different credentials, Storage accounts, and Batch accounts.
 
-### Regions
+#### Service principals
 
-Azure resources can operate across regions, but this incurs additional costs and security requirements. It is recommended to place all resources in the same region. See the [Azure product page on data residency][az-data-residency] for more information.
+An Azure service principal is an identity created for use with applications, hosted services, or automated tools to access Azure resources. A service principal is effectively a user identity with a set of permissions assigned to it. Platform can use an Azure service principal to authenticate and authorize access to Azure Batch for job execution and Azure Storage for data management. By assigning the necessary roles to the service principal, Platform can securely interact with these Azure services. This ensures that only authorized operations are performed during pipeline execution.
```

Comment:
It seems that Platform is assigning the roles to the service principal. "By assigning the necessary roles to the service principal, Platform can.." 
Can we make it more obvious that it's the user who has to set up the roles for the service princpal on their azure account?